release/models/network-instance/openconfig-network-instance-l3.yang

```diff
@@ -23,7 +23,7 @@ module openconfig-network-instance-l3 {
     Layer 3 network instance configuration and operational state
     parameters.";
 
-  oc-ext:openconfig-version "0.11.1";
+  oc-ext:openconfig-version "0.11.0";
```

Comment:
Looks like this is out of date perhaps?

Reply:
Fixed. I suggested we release 1.0.0 of this module since it's something that is implemented.